src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetUpdateMethod.cs

```diff
@@ -421,6 +421,16 @@ public override void ExecuteCmdlet()
             HelpMessage = "Used to make a request conditional for the GET and HEAD methods. The server will only return the requested resources if none of the listed ETag values match the current entity. Used to make a request conditional for the GET and HEAD methods. The server will only return the requested resources if none of the listed ETag values match the current entity. Set to '*' to allow a new record set to be created, but to prevent updating an existing record set. Other values will result in error from server as they are not supported.")]
         public string IfNoneMatch { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Specifies whether resilient VM creation should be enabled on the virtual machine scale set. The default value is false.")]
+        public bool EnableResilientVMCreate { get; set; }
```

Comment:
was there a conclusion as to why these are bool and the others are switch?

Reply:
Creation of resources should only allow True/false. Updating resources show allow true/false/null

Reply:
Create uses Switch because it should allow true/false when the param is present. Updates should allow true/false/null when the param is present.